packages/projects-docs/pages/learn/sandboxes/embedding.mdx

```diff
@@ -36,6 +36,10 @@ This will give you HTML code with an iFrame that you can paste into your content
 
 ![Cloud Sandbox Embed](../images/embed-cloud-sandbox.jpg)
 
+<Callout>
+Although microVM embeds are only available for cloud sandboxes, you can still embed a repository by first importing it as a [synced semplate](/learn/sandboxes/synced-templates).
```

Comment:
Suggestion: MicroVM embeds are a feature of cloud sandboxes. If you have an example from a repository that you want to embed, you can import the repository as a [synced template](/learn/sandboxes/synced-templates) where you can access the code as a cloud sandbox.

Reply:
Thank you! I made some minor tweaks to this suggestion.